srcpkgs/fluent-bit/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'fluent-bit'
+pkgname=fluent-bit
+version=1.7.1
+revision=1
+short_desc="Fluent Bit is an open source Log Processor and Forwarder"
+maintainer="Simon Adameit <simon@simonadameit.com>"
+license="Apache-2.0"
+homepage="https://fluentbit.io/"
+distfiles="https://fluentbit.io/releases/1.7/fluent-bit-${version}.tar.gz"
+checksum=2a870d8b2f3455b83e4660f677c83a9403b75b148c7c33b1b198c63f555e4c16
+
+# Configure build 
+# See:
+# https://docs.fluentbit.io/manual/installation/sources/build-and-install
+build_style=cmake
+export CMAKE_GENERATOR='Unix Makefiles'
+make_cmd=make
+
+hostmakedepends="pkg-config flex bison"
+
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	makedepends+=" musl-fts-devel"
+	configure_args+=" -DCMAKE_C_STANDARD_LIBRARIES=-lfts"
+fi
+
+# Set release flags
+configure_args+=" -DFLB_RELEASE=On"
+configure_args+=" -DFLB_DEBUG=Off"
+configure_args+=" -DFLB_TRACE=Off"
+
+# Enable Features
+configure_args+=" -DFLB_JEMALLOC=On"
```

Comment:
Unbundle `jemalloc` please.

Reply:
And `mbedtls`, too.

Reply:
In other word, I would `rm -rf lib` in `post_extract` and fix their build system to use system libraries.